src/Data/Txt.php

```diff
@@ -40,7 +40,7 @@ public static function encode(mixed $data): string
 	/**
 	 * Helper for converting the value
 	 */
-	protected static function encodeValue(array|string $value): string
+	protected static function encodeValue(array|string|float $value): string
```

Comment:
Where do we pass a `float` at the moment?

Reply:
I couldn't track it down but line 49 suggests we kinda support it... I'd rather allow this type without need than miss a usage and break it